files/beegfs-client.service.d/override.conf

```diff
@@ -0,0 +1,4 @@
+[Service]
```

Comment:
A comment explaining the rationale for these changes will help future you and me.

Reply:
Acknowledged.